core/src/main/java/hudson/console/ModelHyperlinkNote.java

```diff
@@ -27,7 +27,7 @@ public ModelHyperlinkNote(String url, int length) {
 
     @Override
     protected String extraAttributes() {
-        return " class='model-link model-link--float'";
+        return " class='jenkins-table__link model-link model-link--float'";
```

Comment:
This is not a good solution, as this markup is added to the build log as the encoded link is written to it.

This means that only build logs from after a Jenkins update will include these enhancements.

There needs to be a different solution for build logs to adapt to constant style changes than changing the actual log output that is written.

Reply:
Sorry, do you have any better idea to resolve this issue, because I only found this way to solve it ?
Thanks~

Reply:
No idea. If there are CSS classes we can apply to the entire build log and then apply hierarchical rules, that might work.

Otherwise pinging @janfaracik how he imagined this to work.

Reply:
> This means that only build logs from after a Jenkins update will include these enhancements, and old build logs will continue to look broken.

@daniel-beck Does this PR make anything worse? As far as I can tell, it is only an improvement over the status quo, in that new build logs from after a Jenkins upgrade will include this fix. True, a more comprehensive solution could be written, but nobody has volunteered to do that. In fact, the present author even closed this PR! If this change doesn't make anything else worse, but it offers an improvement for new build logs, I think it is worth reopening and merging this PR as-is.

Reply:
@basil Sorry, I have reopen this PR. Thanks~

Reply:
> This is not a good solution, as this markup is added to the build log as the encoded link is written to it.

This statement is wrong. The markup is not part of the serialized form of the object I apologize for getting this wrong and causing a lot of delay here.